### PR TITLE
TW 469 Updates for settings

### PR DESCRIPTION
**Pull request by HeatherStenson**
Changes for the rest of the header updates are happening in https://github.com/postmanlabs/postman-docs/pull/4063

### CODE REVIEW

**Comment by HeatherStenson** [comment]
I'm marking this as a draft for now since there will be some changes as the launch gets closer. I also need to get screenshots from the desktop build once it's available.

**Review by jkonrath-postman — APPROVED** [review]
Looks good to me! I am not sure if we want to go a shorter route on the selection after the select icon (see comment) but let's discuss. Feel free to re-request a review after you get screenshots.

File: src/pages/docs/administration/upgrading.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ Postman v9 provides a consistent, improved API development experience. The Postm
 
 ## Upgrading to Postman v9
 
-You can move from Postman v7 or v8 to Postman v9 by updating the app via **Settings** > **Update** > **Download update**.
+You can move from Postman v7 or v8 to Postman v9 by updating the app. Select the setting icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px">, select **Update**, then select **Download update**.
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/administration/upgrading.md:41]: You're right, this is a good spot to use `>`

File: src/pages/docs/getting-started/navigating-postman.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ To open a new tab, select __+__ in the middle of the screen.
 
 If you open a request and don't edit or send it, then open another request, the second tab replaces the first tab. When the tab is in preview mode, it displays in italics.
 
-You can set whether Postman opens requests in new tabs. Select the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> to open **Settings**. Under **Requests**, select **Always open requests in a new tab** to turn this option on or off.
+You can set whether Postman opens requests in new tabs. Select the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> in the header, then select **General**. Under **Requests**, select **Always open requests in a new tab** to turn this option on or off.
```
Comment (pwitcher) [inline src/pages/docs/getting-started/navigating-postman.md:158]: In Settings > General I see a **Request** heading, but not **Requests**. Also I see **Always open sidebar item in new tab** but not **Always open requests in a new tab**.

File: src/pages/docs/getting-started/navigating-postman.md [hunk 3]
```diff
@@ -185,7 +185,7 @@ To manage your open tabs or to access recently closed tabs, select the more acti
 
 You can also manage your open tabs or open a new tab by right-clicking on an existing tab.
 
-> **Closing unsaved tabs.** You can set whether Postman asks whether you want to save when you close a tab that has changes. Select the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> to open **Settings**. Under **Requests**, select **Always ask when closing unsaved tabs** to turn this option on or off.
+> **Closing unsaved tabs.** You can set whether Postman asks whether you want to save when you close a tab that has changes. Select the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> in the header, then select **General**. Under **Requests**, select **Always ask when closing unsaved tabs** to turn this option on or off.
```
Comment (pwitcher) [inline src/pages/docs/getting-started/navigating-postman.md:188]: I think 'Under **Requests**' should be 'Under **Request**

File: src/pages/docs/getting-started/using-scratch-pad.md [hunk 4]
```diff
@@ -28,9 +28,7 @@ If you sign out or your connection to Postman breaks, you'll see a global banner
 
 <img alt="Scratch Pad global banner" src="https://assets.postman.com/postman-docs/scratch-pad-notice.jpg" width="350px" />
 
-If you are logged in and want to go to the Scratch Pad, select the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> in the header and select **Scratch Pad**.
-
-<img alt="Enter Scratch Pad" src="https://assets.postman.com/postman-docs/scratch-pad-enter.jpg" width="200px" />
+If you are logged in and want to go to the Scratch Pad, select your avatar in the header and select **Scratch Pad**.
```
Comment (pwitcher) [inline src/pages/docs/getting-started/using-scratch-pad.md:31]: I don't see a **Scratch Pad** option when I click my avatar (prod or beta).

File: src/pages/docs/getting-started/settings.md [hunk 5]
```diff
@@ -32,30 +32,42 @@ warning: false
...
+* **Send Postman Token header** - (Recommended) Turn this on to send a random Postman token with an XMLHttpRequest. Sending a random token ensures the receiving server handles one request at a time, even when the requests send with the same parameters. The token can also aid debugging and help you distinguish between requests on the server side.
+* **Retain headers when clicking on links** - When you select a link in a response, Postman creates a new `GET` request with the link URL. Turn this on to keep the headers from the earlier request in the new request. Retaining headers is useful if you mainly access protected resources.
+* **Automatically follow redirects** - Turn this off to prevent requests that return a 3xx series response from automatically redirecting.
+* **Send anonymous usage data to Postman** - Postman gathers basic, anonymous usage data to help with product improvement. Turn this off to stop sending anonymous usage data to Postman.
+
+### User interface
+
```
Comment (pwitcher) [inline src/pages/docs/getting-started/settings.md:61]: I see an additional option under User Interface. On prod, it has **Remove tabs (NEW) User browser tabs to navigate Postman**. On beta, it has **Use the browser tab to navigate postman (NEW)**.

File: src/pages/docs/getting-started/settings.md [hunk 6]
```diff
@@ -32,30 +32,42 @@ warning: false
...
+* **Two-pane view** - By default, Postman displays responses below requests. Turn this on to display the response and request panes side by side.
+* **Show icons with tab names** - Turn this off to hide the icons that appear next to tab names.
+* **Variable autocomplete** - Turn this on to enable autocomplete when typing variable names.
```
Comment (pwitcher) [inline src/pages/docs/getting-started/settings.md:64]: I see another option below **Variable autocomplete** (prod and beta):
**Default documentation editor**

**Comment by jkonrath-postman** [comment]
Closing this without merge - we need to revisit this for a future release.

**Pull request closed without merging**